Resources/Prototypes/Entities/Objects/Devices/pda.yml

```
@@ -1131,7 +1131,7 @@
     - DoorBumpOpener
 
 - type: entity
-  parent: [ BasePDA, BaseSyndicateContraband ]
+  parent: [ BaseSyndiOperativePDA, BaseSyndicateContraband ] #FH edit
```

Tied into other suggestion for NukeOpsPDA
```suggestion
  parent: [ BaseNukeOpsPDA, BaseSyndicateContraband ] #FH edit
```